website/aws.erb

```diff
@@ -2698,6 +2698,9 @@
                                 <li>
                                     <a href="/docs/providers/aws/d/secretsmanager_secret_version.html">aws_secretsmanager_secret_version</a>
                                 </li>
+                                <li>
+                                    <a href="/docs/providers/aws/d/secretsmanager_secret_rotation.html">aws_secretsmanager_secret_rotation</a>
```

Comment:
Nit: The data source and resource should be alphabetized in their respective sections